environment/admin_service.tf

```diff
@@ -49,6 +49,15 @@ locals {
       "protocol": "tcp"
     }],
     "volumesFrom": [],
+    "healthCheck": {
+      "command": [
+        "CMD-SHELL",
+        "curl -f -k https://localhost:443/manage/elb || exit 1"
+      ],
+      "interval": 30,
```

Comment:
Is this supposed to be 90, or is `interval` referring to something different that is mentioned in the PR description?

Reply:
I'm reading this as being three 30 second attempts...but true, couldn't this be one 90 second attempt?

Reply:
Yep, it's three 30 seconds attempts, which is the default configuration from AWS (I had to specify it here because Terraform throws a wobbly otherwise).

The benefit of having more regular attempts is that it can stabilise quicker (in practice, it should be ready in <30 seconds). I was tempted to do a check every five seconds, but it's not really necessary and I don't know what's happening on the AWS side that might not be happy about that.